cli/cmd/tables.go

```diff
@@ -34,6 +36,7 @@ func NewCmdTables() *cobra.Command {
 	}
 	cmd.Flags().String("output-dir", "cq-docs", "Base output directory for generated files")
 	cmd.Flags().String("format", "json", "Output format. One of: json, markdown")
+	cmd.Flags().String("filter", "all", "Filter tables. One of: all, spec")
```

Comment:
```suggestion
	cmd.Flags().String("filter", "", "Filter tables. One of: \"\", spec")
```
the same suggestion as in the original PR

Reply:
I think it's a bit strange to pass `--filter` with an empty value since it's a enum flag. If you drop it will return all tables